docs/source/accelerators/gpu.rst

```diff
@@ -207,23 +207,23 @@ a comma separated list of GPU ids:
     :skipif: torch.cuda.device_count() < 2
 
     # DEFAULT (int) specifies how many GPUs to use per node
-    Trainer(gpus=k)
+    Trainer(accelerator="gpu", devices=k)
 
     # Above is equivalent to
-    Trainer(gpus=list(range(k)))
+    Trainer(accelerator="gpu", devices=list(range(k)))
 
     # Specify which GPUs to use (don't use when running on cluster)
-    Trainer(gpus=[0, 1])
+    Trainer(accelerator="gpu", devices=[0, 1])
 
     # Equivalent using a string
-    Trainer(gpus="0, 1")
+    Trainer(accelerator="gpu", devices="0, 1")
 
     # To use all available GPUs put -1 or '-1'
     # equivalent to list(range(torch.cuda.device_count()))
-    Trainer(gpus=-1)
+    Trainer(accelerator="gpu", devices=-1)
 
 The table below lists examples of possible input formats and how they are interpreted by Lightning.
-Note in particular the difference between `gpus=0`, `gpus=[0]` and `gpus="0"`.
+Note in particular the difference between `devices=0`, `devices=[0]` and `devices="0"`.
 
 +---------------+-----------+---------------------+---------------------------------+
 | `gpus`        | Type      | Parsed              | Meaning                         |
```

Comment:
```suggestion
| `devices`        | Type      | Parsed              | Meaning                         |
```

Reply:
don't go that easy.. fix the formatting xD

Reply:
Hah, figured. done